tensorboard/webapp/widgets/markdown_renderer/markdown_renderer_component_test.ts

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {CommonModule} from '@angular/common';
+import {Component, Input, NO_ERRORS_SCHEMA, ViewChild} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {MarkdownRendererComponent} from './markdown_renderer_component';
+
+@Component({
+  selector: 'testable-markdown-renderer',
+  template: `<markdown-renderer [markdown]="content"> </markdown-renderer>`,
+})
+class TestableComponent {
+  @ViewChild(MarkdownRendererComponent)
+  component!: MarkdownRendererComponent;
+
+  @Input() content!: string;
+}
+
+describe('markdown_renderer/markdown_renderer test', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [TestableComponent, MarkdownRendererComponent],
+      imports: [CommonModule],
+      schemas: [NO_ERRORS_SCHEMA],
+    }).compileComponents();
+  });
+
+  function createComponent(): ComponentFixture<TestableComponent> {
+    return TestBed.createComponent(TestableComponent);
+  }
+
+  function getComponent(fixture: ComponentFixture<TestableComponent>) {
+    return fixture.nativeElement.querySelector('markdown-renderer');
+  }
+
+  it('renders markdown into html', async () => {
+    const fixture = createComponent();
+    fixture.componentInstance.content = '# title';
+    await fixture.detectChanges();
```

Comment:
`detectChanges` does not return promise so it should not be await.

Reply:
Thanks! Please help review agin.